jerry-core/include/jerryscript-core.h

```diff
@@ -98,6 +98,14 @@ typedef enum
   JERRY_FEATURE__COUNT /**< number of features. NOTE: must be at the end of the list */
 } jerry_feature_t;
 
+/**
+ * Option flags for jerry_parse and jerry_parse_function functions.
+ */
+typedef enum
+{
+  JERRY_PARSE_STRICT_MODE = (1 << 0), /**< enable strict mode */
+} jerry_parse_opts_t;
```

Comment:
what about this?
```c
typedef enum
{
  JERRY_PARSE_EMPTY = 0, /**< no flags */
  JERRY_PARSE_STRICT_MODE = (1 << 0), /**< enable strict mode */
} jerry_parse_flag_t;
```
So it will be similar to `jerry_init_flag_t`.

Reply:
I want to phase out the `flag` term from the project. The option bits are much better description.

Reply:
(You can also see that in the snapshot gen/exec flags already landed).

Reply:
And what about adding JERRY_PARSE_EMPTY or JERRY_PARSE_NO_OPTIONS? We should avoid using of literals in call sites of `jerry_parse`.

Reply:
Good idea. NO_OPTS added. I can rename it to NO_OPTIONS if you want.